cmake/FindOpenColorIO.cmake

```diff
@@ -51,7 +51,6 @@ find_library(OCIO_LIBRARY
         libOpenColorIO.so.2.0 # libOpenColorIO.so.2.0 (Linux)
         OpenColorIO.2.0       # libOpenColorIO.2.0.dylib (macOS)
         OpenColorIO_2_0       # OpenColorIO_2_0.lib (Windows)
-        OpenColorIO           # (fallback)
```

Comment:
This seems to still be required for the Windows deps package https://olivevideoeditor.org/deps/dep-win.7z (IIRC, OpenColorIO_2_0 is renamed to just OpenColorIO there)
https://github.com/olive-editor/olive/pull/1489/checks?check_run_id=1929778172

Reply:
New dep package has the DLL under its original name.